### PR TITLE
Feature/flexbox navigation

### PR DESCRIPTION
**Pull request by NafisAli**
## Description

> This version removes grid support and uses flexbox as default in order to reduce complexity and preserve consistency

## Instructions

> Clone or switch to branch feature/flexbox-navigation
> Pull recent updates
> Change your working directory to the root of the project
> Use 'yarn start' to run the project
> Use 'yarn watch:sass' to compile CSS

## Checklist

- [x] Used mixins and variables to avoid unnecessary code duplication .
- [x] SCSS partial name should have `_underscores`.
- [x] Indentation follows style guide (2 spaces soft tabs HTML and CSS).
- [x] Only classes styled (not element selectors).
- [x] All elements have classes.
- [x] Class names are meaningful and consistent.
- [x] Class names follow style guidelines.
- [x] Style rules follow markup order.
- [x] Deleted unused css properties.
- [x] No duplicate class names in SCSS partials.
- [x] No duplicate properties within a rule.
- [x] Validated markup wih w3C Validator.
- [x] Tested and fixed fonts.
- [x] Added `.gitignore` file.
- [x] Fixed browser console errors.
- [x] Used ruler to match (as best as possible) height and width of items.
- [x] Compared front-end with mock up side by side.
- [x] Tested local production `production.css`.
- [x] Tested on mobile, preferably iOS Safari or equivalent Android.
- [x] Tested in Chrome.
- [x] Tested in Firefox.
- [x] Tested in Edge and IE, if available.
- [x] Merged `develop` into your feature branch, multiple times as needed.

You can strike-through items that are not ~~relevant~~.

### CODE REVIEW

File: includes/css/scss/_module-header-brand-dealership.scss [hunk 1]
```diff
@@ -33,23 +23,13 @@
...
 
-	@supports (grid-area: auto) {
-		width: auto;
-		gap: $gap-size;
-	}
-
 	&::before {
 		content: "";
 		height: 2px;
 		width: 3%;
```
Comment (pixelsoup) [inline includes/css/scss/_module-header-brand-dealership.scss]: Please change the `width` to an absolute value. There's no reason the logo should move depending on the width of the `viewport`. **On a side note**, the mobile version of the logo/brand line/dealer name will have to be sorted for mobile, but that's outside the scope of this PR.
```suggestion
		width: 40px;
```

File: includes/css/scss/_module-header-brand-dealership.scss [hunk 2]
```diff
@@ -7,24 +7,13 @@
 		content: "";
 		height: 2px;
 		width: calc((100% - #{$container-width}) / 2);
```
Comment (pixelsoup) [inline includes/css/scss/_module-header-brand-dealership.scss]: No need for `calc` here
```suggestion
		flex: 1;
```

File: includes/css/scss/_module-header-brand-dealership.scss [hunk 3]
```diff
@@ -73,11 +52,6 @@
 	align-items: center;
 	text-decoration: none;
 	margin-right: $gap-size;
```
Comment (pixelsoup) [inline includes/css/scss/_module-header-brand-dealership.scss]: 1. Please change the `$gap-size` value to match the [Volkswagen factory site](https://www.volkswagen.com.au/en.html). Their gap is smaller.
2. Please change your variable name `$gap-size` to something more specific. It's unclear what it's for in `_variables.scss` but it's very specific for one use and can't be repurposed anywhere else on the site. `vw-logo-gap` is better, but feel free to create your own. Please add a short comment above the variable in `_variables.scss` explaining it's purpose.

**Review by pixelsoup — CHANGES_REQUESTED** [review]

File: includes/css/scss/_module-header-brand-dealership.scss (hunk 3 above)
Comment (NafisAli) [reply includes/css/scss/_module-header-brand-dealership.scss]: Resolved in 946a44b and ae05245

**Pull request merged**